.cirrus.yml

```diff
@@ -250,6 +252,92 @@ integration_task:
         package_versions_script: '$GOSRC/$SCRIPT_BASE/logcollector.sh packages'
         golang_version_script: '$GOSRC/$SCRIPT_BASE/logcollector.sh golang'
 
+integration_rootless_task:
+    name: "Integration rootless $DISTRO_NV w/ $STORAGE_DRIVER"
+    alias: integration_rootless
+    only_if: *not_docs
+    depends_on: *smoke_vendor_cross
+
+    matrix:
+        # Running rootless tests on overlay
+        # OVERLAY
+        - env:
+            DISTRO_NV: "${FEDORA_NAME}"
+            IMAGE_NAME: "${FEDORA_CACHE_IMAGE_NAME}"
+            STORAGE_DRIVER: 'overlay'
+            PRIV_NAME: rootless
+        - env:
+            DISTRO_NV: "${PRIOR_FEDORA_NAME}"
+            IMAGE_NAME: "${PRIOR_FEDORA_CACHE_IMAGE_NAME}"
+            STORAGE_DRIVER: 'overlay'
+            PRIV_NAME: rootless
+        - env:
+            DISTRO_NV: "${UBUNTU_NAME}"
+            IMAGE_NAME: "${UBUNTU_CACHE_IMAGE_NAME}"
+            STORAGE_DRIVER: 'overlay'
+            PRIV_NAME: rootless
+
+    gce_instance:
+        image_name: "$IMAGE_NAME"
+
+    # Separate scripts for separate outputs, makes debugging easier.
+    setup_script: '${SCRIPT_BASE}/setup.sh |& ${_TIMESTAMP}'
+    build_script: '${SCRIPT_BASE}/build.sh |& ${_TIMESTAMP}'
+    integration_test_script: '${SCRIPT_BASE}/test.sh integration |& ${_TIMESTAMP}'
+
+    binary_artifacts:
+        path: ./bin/*
+
+    always: &standardlogs
+        audit_log_script: '$GOSRC/$SCRIPT_BASE/logcollector.sh audit'
+        df_script: '$GOSRC/$SCRIPT_BASE/logcollector.sh df'
+        journal_script: '$GOSRC/$SCRIPT_BASE/logcollector.sh journal'
+        podman_system_info_script: '$GOSRC/$SCRIPT_BASE/logcollector.sh podman'
+        buildah_version_script: '$GOSRC/$SCRIPT_BASE/logcollector.sh buildah_version'
+        buildah_info_script: '$GOSRC/$SCRIPT_BASE/logcollector.sh buildah_info'
+        package_versions_script: '$GOSRC/$SCRIPT_BASE/logcollector.sh packages'
+        golang_version_script: '$GOSRC/$SCRIPT_BASE/logcollector.sh golang'
```

Comment:
Please don't re-define the `&standardlogs` anchor.  Simply re-use it by replacing this whole block with the line:

```yaml
    always: *standardlogs
```

Reply:
@cevich, good catch, but I see an inconsistency. The other place where I see `*standardlogs` in that file has a `<<` next to it:
```
    always:
        <<: *standardlogs
```
Is that a trivial syntactic-sugary non-difference, or is it important? To avoid confusing future maintainers, I would suggest sticking with one convention.

Reply:
As long as the second map (dictionary) has no additions, the two are equivalent.  In other words, the second form (with `<<:`) means "copy the anchor value here" and then "add the subsequent new/changed entries".  For example:

```yaml
one: &thing
    foo: abc
    bar: def

two: *thing

three: &newthing
    <<: *thing
    bar: snafu

four: *newthing
```

Means the dictionaries `one` and `two` are identical.  But `three` contains `{foo: abc, bar: snafu}`, and therefor `four` does also.

Reply:
this is fixed in recent push.